corehq/apps/es/client.py

```diff
@@ -1155,6 +1156,12 @@ def scroll(self, *args, **kw):
     def search(self, *args, **kw):
         return self.primary.search(*args, **kw)
 
+    def _get_case_search_sub_index_docs(self, action):
```

Comment:
This function seems to be at the wrong abstraction level. `ElasticMultiplexAdapter` should not need to know about any particular index by name except for the ones it's configured to multiplex via `primary_adapter`/`secondary_adapter`.

Reply:
This also seemed a bit odd to me. Looking at the implementation again it seems like the multiplex adapter doesn't actually call the indexing methods on the primary or secondary but instead uses it's own logic which is why changes need to be applied here as well.

But I do agree with @millerdev that it would be ideal if the multiplex adapter didn't have to know about this. One option could be to have `_render_bulk_action` return a list of actions instead of a single one. That would allow the primary adapter to generate the actions for both the main and sub index.

(I'm OK with the current implementation for the short term)